### ISSUE TITLE
Prevent users from submitting results until set up is complete

### ISSUE THREAD

**Issue by katiealoisi** [CDCgov/prime-simplereport#859]
SR users should not be allowed to conduct tests and submit results until certain minimum requirements are met.

Agreement on the below via Slack on 3/3 (Alicia, Heather, Tim, Katie):

- We will update the facility form in the app to the correct required/optional fields. This will prevent partially completed facilities from being added
- For now, we are still collecting info about at least one facility and creating it as part of our work setting them up
- But we are working towards only setting up their organization and having them add all their own facilities in the app
- Before we can do this, we need to prevent users from starting testing until there is at least one facility, and we need to prevent facilities from being added outside our supported jurisdictions

Outstanding issue: Some states require ordering providers, and some don't. Can Data Hub tell us if ordering provider is required in each state? If so, we could potentially toggle whether we require ordering provider based on the state the facility is in.

**Design for preventing adding a facility outside supported jurisdictions:**
![Screen Shot 2021-03-03 at 4.09.06 PM.png](https://images.zenhubusercontent.com/600201c5b437d97abbef2e2f/caadadc4-e878-47a6-854a-56a867120ec8)

**Design for landing screen to prevent users from starting testing until required steps are taken:**
The "I'm ready to start testing" button should be disabled until the first item is checked off (i.e. they have added at least 1 facility)

![Screen Shot 2021-03-03 at 4.10.05 PM.png](https://images.zenhubusercontent.com/600201c5b437d97abbef2e2f/d1ec3edd-2d39-4a08-90e5-5cb381b45263)

**Comment by katiealoisi** [comment]
@aliciabeckett-gov Can you point us to the minimum requirements that must be met before users can submit data? Or direct us to the right person to ask?

**Comment by katiealoisi** [comment]
Heather to work with Tim and Leslie to turn this into an engineering spec.

**Comment by LGFranklin** [comment]
@vz3 FYI on this --- re: the sign up process. Once we no longer request facility info on sign up, we will need to create a way for users to know what to do to complete the process. Heather created a design here to work from. 
In the notes you will also see a reference to ordering provider. At current, North Dakota is the only state that does NOT require this.

**Comment by timothybest-usds** [comment]
closing in favor of https://github.com/CDCgov/prime-simplereport/issues/1998
